content/en/about/feature-stages/index.md

```diff
@@ -55,12 +55,10 @@ Below is our list of existing features and their current phases. This informatio
 |-------------------|-------------------
 | [Prometheus Integration](/docs/tasks/observability/metrics/querying-metrics/) | Stable
 | [Local Logging (STDIO)](/docs/tasks/observability/mixer/logs/collecting-logs/) | Stable
```

Comment:
this can be removed now

Reply:
Removed in an earlier PR. Picked up removal in a rebase.